README.md

```diff
@@ -18,8 +18,7 @@
 ## Build requirements
 
 * Mac OS X or Linux
-* Java 11.0.8, 64-bit
-* Python 2.6+ (for running with the launcher script)
+* Java 11.0.7, 64-bit
```

Comment:
Do we want to say `11.0.7+`?

Reply:
probably... but you also need to clarify that it needs to be Java 11..

Reply:
It should build fine with newer versions, since we set the target version in the POM. We do the run the main Maven build against JDK 15 in the CI.

Reply:
We know that the code compiles with Java 15 and I believe it works, but strictly speaking, we do not test this.

Reply:
These are the build requirements. We have CI tests that verify it builds using 15.

Reply:
My old company came to the same conclusion about Javas before 11.07. Adapting what I wrote for them then, how about the following? Clearly, I prefer being very explicit about what is supported:

- Java 11.0.7 or a later Java 11 version. OpenJDK, Oracle Java, and Azul Zulu are supported.